src/components/common/privacy.js

```diff
@@ -1,14 +1,12 @@
 import React from 'react';
+import { PRIVACY_CONTENT } from '../../utils/constants';
 
 export const PrivacyComponent = () => {
   return (
     <div className="mx-auto max-w-7xl text-center rounded-lg mb-10 border border-transparent bg-gradient-to-r from-purple-600 to-indigo-600 bg-origin-border px-5 py-10 text-base font-medium text-white shadow-sm">
-      <h2 className="font-semibold text-3xl">We Take Your Privacy Seriously</h2>
+      <h2 className="font-semibold text-3xl">{PRIVACY_CONTENT.title}</h2>
```

Comment:
why do we need constant for this?
this is related to this particular component, we are not using this text anywhere else
I don't think it makes sense to assign it to a constant